queue/queue.go

```diff
@@ -0,0 +1,75 @@
+package queue
+
+import (
+	"sync"
+	"time"
+)
+
+// Build is a request waiting for a worker
+type Build struct {
+	Pipeline string `json:"pipeline"`
+	Manifest string `json:"manifest"`
+}
+
+// Job is an image build already in progress
+type Job struct {
+	UUID     string `json:"uuid"`
+	WorkerID string `json:"worker-id"`
```

Comment:
Our model currently does not contain a worker objcet/ID, so I suggest dropping this for now. We almost certainly want to introduce it, but we should do it at the same time in the API I think, once we have discussed how it should look like.